src/Libraries/RevitNodes/GeometryConversion/ProtoToRevitFace.cs

```diff
@@ -0,0 +1,154 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Autodesk.Revit.DB;
+using Autodesk.DesignScript.Runtime;
+using Edge = Autodesk.DesignScript.Geometry.Edge;
+using Face = Autodesk.DesignScript.Geometry.Face;
+using ProtoPt = Autodesk.DesignScript.Geometry.Point;
+using Solid = Autodesk.DesignScript.Geometry.Solid;
+using Surface = Autodesk.DesignScript.Geometry.Surface;
+
+namespace Revit.GeometryConversion
+{
+   [SupressImportIntoVM]
+   public static class ProtoToRevitSolid
+   {
+     /// <summary>
+     /// this method attmempts to construct a BRep from a closed solid.
+     /// </summary>
+     /// <param name="sol"></param>
+     /// <param name="performHostUnitConversion"></param>
+     /// <returns></returns>
+       public static GeometryObject ToRevitType(Solid sol,
+          bool performHostUnitConversion = true)
+       {
+           try
+           {
+               return ToRevitType(sol, performHostUnitConversion, BRepType.Solid);
+           }
+           catch
+           {
+               return null;
```

Comment:
Wondering if it would be nice to log this as a failure  in the Dynamo console with a message like "failed to convert geometry into Revit BRep" or something on those lines.